### ISSUE TITLE
Output resource identifier in Fetch Diagnostics

### ISSUE THREAD

**Issue by serain** [cloudquery/cloudquery#473]
### Is your feature request related to a problem? Please describe.
When encountering errors, i.e. 403s, while ingesting resources, the cloudquery Fetch Diagnostics don't include a unique resource identifier.

This tells us there's a problem ingesting some resources without telling us on which resources the problems were encountered, making debugging difficult.

example:
```
	Summary: table[aws_s3_bucket_lifecycles] resolver ignored error. Error: operation error S3: GetBucketLifecycleConfiguration, https response error StatusCode: 403, RequestID: xxxx, HostID: xxxx, api error AccessDenied: Access Denied
Resource: apigateway.rest_apis Type: Resolving Severity: Error
	Summary: operation error API Gateway: GetModelTemplate, https response error StatusCode: 400, RequestID: xxxx, BadRequestException: Cannot generate mapping template because model schema is missing a 'type' or '$ref' property
```

### Describe the solution you'd like
Would be great to know on which resources the issues happened, where possible (see examples, where cloudquery should already have the info to tell us which S3 bucket and which API Gateway are concerned)

**Comment by roneli** [comment]
Hi @serain, thanks for the feature request!

We started working on the feature, hopefully it will be available by end of the week / start of next week.

**Comment by roneli** [comment]
Hi @serain this feature should be available in v0.20.4, v0.10.5 of cloudquery and aws provider respectively.
